data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -39,9 +38,8 @@ class TokenRepository(
         })
     }
 
-    //FIXME: use client gateway (grouping and sorting will be done on the backend side)
     suspend fun loadCollectiblesOf(safe: Solidity.Address): List<Collectible> =
-        transactionServiceApi.loadCollectibles(safe.asEthereumAddressChecksumString())
+        gatewayApi.loadCollectibles(safe.asEthereumAddressChecksumString())
             .asSequence()
             .groupBy {
                 it.address
```

Comment:
Does client gw perform grouping?

Reply:
Nope. Just caching. There is no change in the `DTO` classes. The response is the same as from the TX service, as specified in the ticket.